health_monitor/src/health_monitor.cpp

```diff
@@ -72,7 +72,7 @@ namespace health_monitor
         start_up_timestamp_ = ros::Time::now().toNSec() / 1e6;
         start_time_flag_=ros::Time::now();
 
-        pnh_->setSystemAlertCallback([&](const cav_msgs::SystemAlertConstPtr& msg) -> void {
+        ros::CARMANodeHandle::setSystemAlertCallback([&](const cav_msgs::SystemAlertConstPtr& msg) -> void {
```

Comment:
I received an error when I tried to build the health monitor package:

 error: cannot call member function 'void ros::CARMANodeHandle::publishSystemAlert(const SystemAlert&)' without object
                 ros::CARMANodeHandle::publishSystemAlert(new_msg);
                                                                 ^